packages/app-store/jitsivideo/components/InstallAppButton.tsx

```diff
@@ -0,0 +1,18 @@
+import type { InstallAppButtonProps } from "@calcom/app-store/types";
+
+import useAddAppMutation from "../../_utils/useAddAppMutation";
+
+export default function InstallAppButton(props: InstallAppButtonProps) {
+  const mutation = useAddAppMutation("jitsi_video");
+
```

Comment:
I think here we need to call a trpc.invalidate query on the list of installed apps, so the new mutation changes kick in in the query and we don't have ghosting states in between

Reply:
The endpoint to checks for installed apps is not on trpc as of now. We may address your feedback in a follow up @agustif